docs/UserGuide.md

```diff
@@ -256,20 +257,29 @@ Lists all the entries in the user's spending list.
 
 **Format:** `list spendings`
 
-The user will then be prompted to select a time range.
-Only entries that are within the time range will be displayed.
-If option 1 (all) is chosen, the user will then be asked if all spending statistics should be displayed (case insensitive).
+The user will then be prompted to select a time range from the following options:
+1. All
+2. This week
+3. Past 2 weeks
+4. This month
+
+Only entries that are dated within the selected time range will be displayed. 
+<br> E.g. If today is 20 November 2024 (Wednesday) and the user chooses option 2 (this week),  only spending entries 
+from the current week's Monday (18 November 2024) to Sunday (24 November 2024) will be shown.
+
+If option 1 (all) is chosen, the user will then be asked if all spending statistics should be displayed 
+(Y/N is case insensitive).
 
 **Example Input and Output**
 
 **Input:** `list spendings`
 ```
-    ____________________________________________________________
-    Select time range:
-    [1] All
-    [2] Weekly
-    [3] Biweekly
-    [4] Monthly
+	____________________________________________________________
+	List spending entries within:
+	[1] All
+	[2] This week
+	[3] Past 2 weeks
+	[4] This month
```

Comment:
`Past 2 weeks` can be abit confusing also since it can mean the past 14 days instead from last monday to today. Maybe use `List spending entries for` and `[3] last and this week`?

Reply:
Okay, fixed